lib/environment_setup.rb

```diff
@@ -52,6 +53,10 @@ def check_changed
     end
   end
 
+  def setup_site_name
+    update "STOCKAID_SITE_NAME", prompt: "What is your site's name?"
```

Comment:
I considered this, however I opted against it since `config/initializers/site_name.rb` will take an empty value and default it to `StockAid` already :-)

I even started updating `update` with a way to provide a default, but I backed it out once I realized I didn't REALLY need it....